.vscode/launch.json

```diff
@@ -199,21 +219,44 @@
             "console": "internalConsole"
         },
         {
-            "name": "server: RQ - scheduler",
+            "name": "server: RQ - quality reports",
```

Comment:
Why we don't add new workers in `server: debug` vscode compound?

Reply:
Yes, I can add it.